src/fides/api/tasks/__init__.py

```diff
@@ -11,6 +19,7 @@
 MESSAGING_QUEUE_NAME = "fidesops.messaging"
 PRIVACY_PREFERENCES_QUEUE_NAME = "fides.privacy_preferences"  # This queue is used in Fidesplus for saving privacy preferences and notices served
 
+NEW_SESSION_RETRIES = 5
```

Comment:
How did we come up with this int?

Reply:
I wanted to retry for 30 seconds and with an exponential backoff, 5 attempts (first call + 4 retries) gives us that number (2 + 4 + 8 + 16)